.github/workflows/sycl-windows-build.yml

```diff
@@ -101,16 +120,24 @@ jobs:
         cmake --build build --target sycl-toolchain
     - name: check-llvm
       if: always() && !cancelled() && contains(inputs.changes, 'llvm')
+      shell: bash
       run: |
+        if [[ ${{inputs.compiler}} == 'icx' ]]; then
```

Comment:
Similarly to CMake options, that's better be handled elsewhere (LIT feature + `REQUIRES`?)

Reply:
yeah i think its reasonable to add a LIT feature but hopefully I can do this in a follow-up commit

Reply:
made https://github.com/intel/llvm/issues/16363 for follow up